embedding/csharp/FlutterApplication.cs

```diff
@@ -64,7 +64,7 @@ protected override void OnCreate()
                 switches_count = (uint)switches.Length,
```

Comment:
Note: The C# equivalent of `size_t` in C is `nuint` (available in C# 9.0) or `IntPtr` (technically identical), but somehow marshaling from `uint` still works fine on arm64 devices. I don't want to increase the C# and .NET SDK version requirement right now so I'll leave as it is.